Rules/rules.css

```diff
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Patrick+Hand&family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Mali:wght@400&family=Patrick+Hand&family=Roboto&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Mali", cursive;
+}
+
+:root {
+  --bgcolor: #edf2f4;
+  --gray: #8d99ae;
+  --primary-color: #2b2d42;
+  --white: #ffffff;
+  --red: #ef233c;
+}
```

Comment:
You can create a common utility file and import it once wherever needed. No need of writing these properties in all style files.